### ISSUE TITLE
PHP warning

### ISSUE THREAD

**Issue by aamasri** [iCalcreator/iCalcreator#75]
kigkonsult/icalcreator version: 2.27.19
PHP Version: 7.3.5-1+ubuntu18.04.1

```
A WARNING occurred on line 920 in file: ..../vendor/kigkonsult/icalcreator/src/Util/RecurFactory.php.

"continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?
```

**Comment by aamasri** [comment]
Let me know if you'd like my to fix and submit a PR?

**Comment by iCalcreator** [comment]
Yes, a continue/break confusion, most likely a missing break.
I'll look into the issue and opt. extend the tests.

**Comment by iCalcreator** [comment]
Fixed in 2.27.20.

**Comment by aamasri** [comment]
> Fixed in 2.27.20.

👍 Thank you!
